### DISCUSSION TITLE
How to start contributing as a newbie

### DISCUSSION THREAD

**Discussion by tirstex777** [cncf/mentoring#719] [Q&A] [answered]
i'm new comer and i really want to learn and contribute ....can anyone please guide me or point me to some resources where i can start from

**Comment by nate-double-u** [comment, marked as answer]
Hi @tirstex777!

If you want to be a Cloud Native contributor, I suggest starting by checking out the [CNCF Contributors](https://contribute.cncf.io/) page.

If you're interested in being a mentee, you can also take a look around this repo and see what opportunities there are. Some projects listed here have issues marked "Good first issue" which you can take a look at. It's good to communicate and work with projects that you're interested in doing a mentorship with.
